Configurations/release-move-tag.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+# Convenience script to automatically commit Package.swift after updating the checksum and move the latest tag
+latest_git_tag=$(git describe --abbrev=0) # gets the latest tag name
+commits_since_tag=$(git rev-list ${latest_git_tag}.. --count)
+if [ "$commits_since_tag" -gt 0 ]; then
+    # If there have been commits since the latest tag, it's highly likely that we did not intend to do a full release
+    echo "WARNING: $commits_since_tag commit(s) since tag '$latest_git_tag'. Did you tag a new version?"
+    echo "Package.swift has not been committed and tag has not been moved."
+else
+    # TODO: add sanity check to see if version is actually being updated or not?
+    read -p "Do you want to commit changes to Package.swift and force move tag '$latest_git_tag'? (required for Swift Package Manager release) " -n 1 -r
```

Comment:
The script asks for user approval here. It would be entirely feasible to move the tag automatically when there have been no commits. Indeed, not much damage can be done as if there are actually no changes to Package.swift (e.g., we're running the script on its own for some reason), `git add Package.swift` fails with an error and nothing else is done.
Should I remove interaction in favour of automaticity? What do you think?

Reply:
Let's keep a check for now